spancontext.go

```diff
@@ -57,10 +57,6 @@ type DatabaseSpanContext struct {
 
 // DestinationServiceSpanContext holds destination service span span.
 type DestinationServiceSpanContext struct {
-	// Name holds a name for the destination service, which may be used
-	// for grouping and labeling in service maps.
-	Name string
```

Comment:
We can't remove this, that would be a breaking API change. Just mark it as deprecated and don't require it to be set.